### PR TITLE
Custom Properties and Web Components article

### PR DESCRIPTION
**Pull request by rachelandrew**
Article by David Darnes on work at Nordhealth.

### CODE REVIEW

File: src/site/content/en/blog/custom-properties-web-components/index.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+.n-tab-group-list::after {
+  /* ... */
+  padding-inline-end: var(--n-tab-group-padding);
+}
+```
+
+<figcaption>The tab group padding contextual Custom Property being used in multiple places within the component code.</figcaption>
+</figure>
+  
+And second, it makes component state and variation changes really clean— it's only the custom property that needs to be changed to update all those properties when, say, you're styling a hover or active state or, in this case, a variation.
```
Comment (daviddarnes) [inline src/site/content/en/blog/custom-properties-web-components/index.md:187]:
```suggestion
And second, it makes component state and variation changes really clean—it's only the custom property that needs to be changed to update all those properties when, say, you're styling a hover or active state or, in this case, a variation.
```

Just removing the space after the —

**Comment by rachelandrew** [comment]
@daviddarnes can you sign the CLA as mentioned in the checks?

**Comment by daviddarnes** [comment]
@rachelandrew done!

File: src/site/content/en/blog/custom-properties-web-components/index.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
+---
+layout: post
+title: How Nordhealth uses Custom Properties in Web Components
+subhead: The benefits of using Custom Properties in design systems and component libraries.
+authors:
+  - daviddarnes
+description: The benefits of using Custom Properties in design systems and component libraries.
+date: 2022-08-23
```
Comment (rachelandrew) [inline src/site/content/en/blog/custom-properties-web-components/index.md:8]:
```suggestion
date: 2022-08-17
```

File: src/site/content/en/blog/custom-properties-web-components/index.md [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+</figure>
+
+While it may be argued that this method isn't truly "private" we still think this is a rather elegant solution to a problem we were worried about. When we have the opportunity, we'll be tackling this in our components so our development team has more control over component usage while still benefiting from the guardrails we have in place. 
+
+I hope you found this insight into how we use Web Components with CSS Custom Properties useful. Please let us know what you think and if you decide to use any of these methods in your own work, you can find me on Twitter [@DavidDarnes](https://twitter.com/DavidDarnes?ref_src=twsrc%5Etfw). You can also find Nordhealth [@NordhealthHQ](https://twitter.com/nordhealthhq?lang=en) on Twitter, as well as the rest of my team, who have worked hard on bringing this design system together and executing the features mentioned in this article: [@Viljamis](https://twitter.com/viljamis?lang=en), [@WickyNilliams](https://twitter.com/wickynilliams?lang=en) and [@eric_habich](https://twitter.com/eric_habich).
```
Comment (daviddarnes) [inline src/site/content/en/blog/custom-properties-web-components/index.md:311]:
```suggestion
I hope you found this insight into how we use Web Components with CSS Custom Properties useful. Let us know what you think and if you decide to use any of these methods in your own work, you can find me on Twitter [@DavidDarnes](https://twitter.com/DavidDarnes?ref_src=twsrc%5Etfw). You can also find Nordhealth [@NordhealthHQ](https://twitter.com/nordhealthhq?lang=en) on Twitter, as well as the rest of my team, who have worked hard on bringing this design system together and executing the features mentioned in this article: [@Viljamis](https://twitter.com/viljamis?lang=en), [@WickyNilliams](https://twitter.com/wickynilliams?lang=en) and [@eric_habich](https://twitter.com/eric_habich).
```

**Pull request merged**